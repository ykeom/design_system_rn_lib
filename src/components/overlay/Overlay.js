import React, { Component } from 'react';
import { View, StyleSheet, Text } from 'react-native';

const styles = StyleSheet.create({
  root: {},
});

class Overlay extends Component {
  render() {
    return (
      <View style={styles.root}>
        <Text>overlay</Text>
      </View>
    );
  }
}

export default Overlay;
