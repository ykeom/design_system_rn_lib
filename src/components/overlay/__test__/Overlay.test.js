import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';
import Overlay from '../Overlay';

describe('Overlay', () => {
  it('should render', () => {
    const component = shallow(<Overlay />);

    expect(component.length).toBe(1);
    expect(toJson(component)).toMatchSnapshot();
  });
});
