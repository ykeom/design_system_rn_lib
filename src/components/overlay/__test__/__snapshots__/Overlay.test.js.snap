// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Overlay should render 1`] = `
<View
  style={Object {}}
>
  <Text>
    overlay
  </Text>
</View>
`;
